src/actors/symbolication.rs

```diff
@@ -1304,21 +1304,28 @@ impl SymbolicationActor {
                                     object_info_from_minidump_module(object_type, code_module)
                                         .into();
 
-                                let status = unwind_statuses
-                                    .get(&code_module.id()?)
-                                    .cloned()
-                                    .unwrap_or(ObjectFileStatus::Unused);
+                                let module_id: Option<CodeModuleId> = code_module.id();
```

Comment:
This type annotation is redundant, as this can be inferred statically.

Reply:
Yes indeed.  I added it because I find it improves readability.  I like being able to read code without needing rust-analyzer...  But the code matches on it right away so maybe it's obvious enough if you don't care about what type the id itself actually is.